src/vault/vault.ts

```diff
@@ -128,4 +128,13 @@ export class Vault {
     const dataKey = await this.decryptDataKey({ keys: decoded.keys });
     return decrypt(decoded, dataKey.key);
   }
+
+  // Deprecated methods
+  createSecret = this.createObject;
+  listSecrets = this.listObjects;
+  listSecretVersions = this.listObjectVersions;
+  readSecret = this.readObject;
+  describeSecret = this.describeObject;
+  updateSecret = this.updateObject;
+  deleteSecret = this.deleteObject;
```

Comment:
suggestion: Since these are deprecated, it would be beneficial to mark them as such with a `@deprecated` tag.

```ts
/**
* @deprecated Use <OTHER_METHOD> instead
*/
```

![capture_20250327_083550](https://github.com/user-attachments/assets/ef04c4f0-76ef-45fa-b0bd-f162d0843235)